examples/borer-circe-migration/src/main/scala/btest/Transcoding.scala

```diff
@@ -0,0 +1,46 @@
+package pl.marconzet
```

Comment:
Also... the folder name `borer-circe-migration` is heavily misleading, it suggests that some stuff is migrated **between Borer and Circe** (which wouldn't be of much value)

Reply:
Maybe just `examples/event-migration/`? and package structure will then show which libraries are considered (also, what if examples for a third library are added... `borer-circe-avro-migration`? ;) )